src/TimelineItem.cc

```diff
@@ -30,7 +30,7 @@
 #include "TimelineViewManager.h"
 
 static const QRegExp URL_REGEX("((?:https?|ftp)://\\S+)");
-static const QString URL_HTML = "<a href=\"\\1\" style=\"color: #333333\">\\1</a>";
+static const QString URL_HTML = "<a href=\"\\1\">\\1</a>";
```

Comment:
Probably we need to specify a color for the links also. The default blue might be too hard for some themes.

Reply:
I found that the link color becomes a reasonable one chosen by the theme if this is not set. Additionally, it was a pain trying to get links inside a `TimelineItem` to get a color set by the stylesheet; there is no way to set the color of links inside an existing `QLabel`, so you would have to generate the content of all the `QLabel`s again with the new color set for every link when the link color changes. And properties defined in the stylesheet are set only *after* the object's initializer is called, which means the `TimelineItem` can not yet know its preferred link color at the time that all the `QLabel`s are being created.